### PR TITLE
Added SM64 Assets

### PR DESCRIPTION
**Pull request by Lang-Kasempo**
This is what I can do when I'm not lazy :^
Also 
*dances at the rhythm of Super Mario64 Slide Music*

### CODE REVIEW

File: assets/SM64/sm64config.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+key_cup 0148 ffff ffff 
+key_cdown 0150 ffff ffff 
+key_cleft 014b ffff ffff 
+key_cright 014d ffff ffff 
+key_stickup 0011 ffff ffff 
+key_stickdown 001f ffff ffff 
+key_stickleft 001e ffff ffff 
+key_stickright 0020 ffff ffff 
+stick_deadzone 15
+rumble_strength 50
```
Comment (Lugsole) [inline assets/SM64/sm64config.txt:27]: I thought you wanted this to be zero.

File: assets/SM64/Super Mario 64.desktop [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Icon=/usr/share/SM64/SM64Icon.png
+Name=Super Mario 64
+Exec=/usr/share/SM64/sm64
+Terminal=false
+Hidden=false
+Categories=Games
+Name[es_ES]=Super Mario 64
```
Comment (Lugsole) [inline assets/SM64/Super Mario 64.desktop:9]: Did you want this to have a special name for "Spanish (Spain)"? otherwise you could remove this line

**Comment by Lugsole** [comment]
Also squash the commits to one.

File: assets/SM64/Super Mario 64.desktop (hunk 2 above)
Comment (Lang-Kasempo) [reply assets/SM64/Super Mario 64.desktop:9]: Actually, for some reason Arronax created that line ^^"

**Pull request merged**